src/test/java/nextstep/subway/domain/StationTest.java

```diff
@@ -0,0 +1,40 @@
+package nextstep.subway.domain;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+import org.springframework.boot.test.autoconfigure.orm.jpa.TestEntityManager;
+import org.springframework.test.annotation.DirtiesContext;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@DataJpaTest
+@DirtiesContext(classMode = DirtiesContext.ClassMode.AFTER_EACH_TEST_METHOD)
+public class StationTest {
+    @Autowired
+    private StationRepository stationRepository;
+    @Autowired
+    private TestEntityManager entityManager;
+
+
+    @DisplayName("지하철역 생성 테스트")
+    @Test
+    void 지하철역생성_성공() {
+        // given
+        Station sinsa = new Station("신사역");
+
+        // when
+        stationRepository.save(sinsa);
+        flushAndClear();
+        Station station = stationRepository.findById(1L).get();
```

Comment:
```suggestion
        assertThat(stationRepository.findById).{method}
```
바로 get하는 것 보단 findById 자체를 감싸는 것도 좋을 것 같아요!